gazebo_ros/scripts/spawn_model.py

```diff
@@ -0,0 +1,312 @@
+#!/usr/bin/env python3
```

Comment:
How about renaming this script to `spawn_entity`, like we did with the service? It should work not only for models, but also for lights, as explained in the [migration page](https://github.com/ros-simulation/gazebo_ros_pkgs/wiki/ROS-2-Migration:-Spawn-and-delete#summary).